component/src/test/java/org/wso2/extension/siddhi/io/nats/source/NATSSourceTestCase.java

```diff
@@ -0,0 +1,892 @@
+                + "destination='nats-test10', "
+                + "client.id='nats-source-test10-siddhi', "
+                + "bootstrap.servers='nats://localhost:5223', "
+                + "cluster.id='test-cluster'"
+                + ")"
+                + "define stream inputStream (name string, age int, country string);"
+                + "@info(name = 'query1') "
+                + "from inputStream "
+                + "select *  "
+                + "insert into outputStream;";
+
+        SiddhiAppRuntime executionPlanRuntime = siddhiManager.createSiddhiAppRuntime(siddhiApp);
+        executionPlanRuntime.start();
+        Thread.sleep(1000);
+
+        Assert.assertTrue(appender.getMessages().contains("Error while connecting to NATS server at destination: "
+                + "nats-test10"));
+        siddhiManager.shutdown();
+    }
+
+    /**
+     * Tests the ability to subscribe to a NATS subject from given sequence number.
+     */
+    @Test(dependsOnMethods = "testIncorrectNatsServerUrl")
+    public void testNatsSequenceSubscribtion() throws InterruptedException, TimeoutException, IOException {
+        ResultContainer resultContainer = new ResultContainer(6,3);
+        NATSClient NATSClient = new NATSClient("test-cluster", "nats-source-test1",
+                "nats://localhost:" + port);
+        NATSClient.connect();
+
+        NATSClient.publish("nats-test11","<events><event><name>JAMES</name><age>22</age>"
+                + "<country>US</country></event></events>");
+        NATSClient.publish("nats-test11","<events><event><name>MIKE</name><age>30</age>"
+                + "<country>GERMANY</country></event></events>");
+        NATSClient.publish("nats-test11","<events><event><name>JHON</name><age>25</age>"
+                + "<country>US</country></event></events>");
+        NATSClient.publish("nats-test11","<events><event><name>ARUN</name><age>52</age>"
+                + "<country>GERMANY</country></event></events>");
+        NATSClient.publish("nats-test11","<events><event><name>ALICE</name><age>32</age>"
+                + "<country>US</country></event></events>");
+        NATSClient.publish("nats-test11","<events><event><name>BOP</name><age>28</age>"
+                + "<country>GERMANY</country></event></events>");
+        NATSClient.publish("nats-test11","<events><event><name>JAKE</name><age>52</age>"
+                + "<country>US</country></event></events>");
+        NATSClient.publish("nats-test11","<events><event><name>RAHEEM</name><age>47</age>"
+                + "<country>GERMANY</country></event></events>");
+        NATSClient.publish("nats-test11","<events><event><name>JANE</name><age>36</age>"
+                + "<country>US</country></event></events>");
+        NATSClient.publish("nats-test11","<events><event><name>LAKE</name><age>19</age>"
+                + "<country>GERMANY</country></event></events>");
+        Thread.sleep(3000);
+
+
```

Comment:
Remove unnecessary new lines. Check other places as well

Reply:
Resolved with the commit 7e05e862337f827854551f7cdea54c88c72623e4.